scripts/presubmit.sh

```diff
@@ -128,8 +132,14 @@ main() {
           ${goflags} "$d"
     done | xargs -I '{}' -P ${GO_TEST_PARALLELISM:-10} bash -c '{}'
 
-    [[ ${coverage} -eq 1 ]] && \
-      cat /tmp/trillian_profile/*.out > /tmp/coverage.txt
+    cat /tmp/trillian_profile/*.out > /tmp/coverage.txt
+  fi
+  else
+      echo go test \
```

Comment:
While this will indeed be fast, that's because it won't actually run the tests. Need to delete the "echo" at the start.

Reply:
hehe, whoops!